package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "app",
+  "private": "true",
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.7",
+    "react-redux": "^8.1.3",
+    "react-router-dom": "^6.16.0",
+    "redux": "^4.2.1",
+    "redux-thunk": "^2.4.2"
+  }
```

Comment:
- [ ] react-dom and babel-loader should be part of these dependencies, it means you have not yet added react library, kindly add it too.